snapcraft/templates/kernel/ubuntu_kernel_build_kernel_debs.sh.inc.j2

```diff
@@ -0,0 +1,98 @@
+# --- BEGIN kernel source build ---
```

Comment:
This directory is the wrong place for these jinja templates. The `templates` directory is for jinja templates for the `init` command. You could probably put it directly in an `ubuntu_kernel` directory under `parts/plugins` though.